client/webserver/api.go

```diff
@@ -701,7 +721,13 @@ func (s *WebServer) apiWithdraw(w http.ResponseWriter, r *http.Request) {
 		s.writeAPIError(w, fmt.Errorf("no wallet found for %s", unbip(form.AssetID)))
 		return
 	}
-	coin, err := s.core.Withdraw(form.Pass, form.AssetID, form.Value, form.Address)
+	pass, err := s.resolvePass(form.Pass, r)
+	if err != nil {
+		s.writeAPIError(w, fmt.Errorf("password error: %w", err))
+		return
+	}
+	defer zero(pass)
+	coin, err := s.core.Withdraw(pass, form.AssetID, form.Value, form.Address)
```

Comment:
Sort of unrelated, but looked like an oversight. We can used a cached password here same as other places. Could even hide the password box if it is cached. Unless this was intended to always need a password?